### DISCUSSION TITLE
Improved CSV import

### DISCUSSION THREAD

**Discussion by WolfgangFahl** [SmartDataAnalytics/OpenResearch#230] [Ideas]
To speed up entry or relevant data for event series and improved CSV "round trip" could be very valuable. This way spreadsheet-tools such as Excel might be used for quick data updates.

Take the series https://confident.dbis.rwth-aachen.de/or/index.php?title=FCT as an example. Manually editing all entries in a column takes some 50 minutes. With a CSV round trip this time could be reduced to a few minutes e.g. since formulas might be used and cut&paste of entries is much easier.

**Comment by WolfgangFahl** [comment]
#232 has an example that would benefit from the functionality above. A good start would be to merge the Proceedings Title Parser and dlpconf tools to a single Scientific Event Corpus Tool (SEC?) . For a start it is sufficient that the dblp conf tool uses the event_all database of PTP and the PTP API. Later the search interface can be migrated. We'd love to have an API where each event series and event has it's own restful endpoint. To do so we need to be able to uniquely identify an event series or event. Since most series don't have PIDs yet this would be a good starting point to get PIDs for the once we are interested in.  As long as we do not have PIDs we'll use Pseudo-PIDs that is the event identifiers of our source systems e.g. wikidata, dblp, wikcfp, GND,  or if all fails the acronym.

**Reply by MusaabKh** [reply]
![Csv Round Trip](https://user-images.githubusercontent.com/26063969/119109143-5c0a9900-ba3a-11eb-89d1-dcbbba97962f.png)

**Reply by tholzheim** [reply]
Situation: Event Series 3DUI is given
Action: Edit it with an external csv editor
Expected result: Corresponding Event pages such as 3DUI 2014 and 3DUI 2020 are changed. New pages created if needed

**Reply by tholzheim** [reply]
Possible csv output:
```csv
State,Country,Ordinal,End date,Submission deadline,pageTitle,Acronym,Series,has Proceedings Link,Start date,Homepage,Type,City,Title 
South Carolina,USA,,2016/03/20,,3DUI 2016,3DUI 2016,3DUI,,2016/03/19,,Conference,Greenville,IEEE Symposium on 3D User Interfaces 
,USA,,2020/03/26,2019/09/03,3DUI 2020,3DUI 2020,3DUI,,2020/03/22,http://ieeevr.org/2020/,Symposium,Atlanta,IEEE Symposium on 3D User Interfaces 
,USA,15,2010/03/21,,3dui 2010,3DUI 2020,3DUI,https://www.computer.org/csdl/proceedings/3dui/2010/12OmNAsTgX7,2010/03/20,http://ieeevr.org/2020/,Symposium,Waltham,IEEE Symposium on 3D User Interfaces 
```
